openshift/f8a-notification.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1     
+kind: Template
+labels:
+  template: f8a-notification-scheduler
+metadata:
+  name: f8a-notification
```

Comment:
Please rename to `f8a-notification-scheduler`

Reply:
Done.